modules/passkey/test/4337/experimental/Safe4337Module.spec.ts

```diff
@@ -175,78 +174,6 @@ describe('Safe4337Module', () => {
         expect(await safeInstance.getOwners()).to.deep.equal([signerAddress])
       })
     })
-
-    describe('executeUserOp - existing account', () => {
```

Comment:
This test wasn't actually removed, just moved to `modules/passkey/test/4337/SafeWebAuthnSigner.spec.ts` as it is unrelated to the launchpad and was in the wrong place to begin with.

Conceptually, there are no "executeUserOp - existing account" tests for the launchpad, as it is only ever used for account creation.